### DISCUSSION TITLE
Abort blur event (element loses focus) on safelink site

### DISCUSSION THREAD

**Discussion by Recehan-Slayer** [realodix/AdBlockID#734] [General]
**Jenis iklan** (hapus yang tidak perlu)
- Boost timer

**Sample URL**
https://semawur.com/JrZ9GnIVp

**Penjelasan Lebih Lanjut**
`semawur.com##+js(nano-sib)`

bro klo shorteners yg kadang mendeteksi out of focus masuk filter adblockID jg ga

**Comment by realodix** [comment]
**Related** https://github.com/realodix/AdBlockID/issues/91

---

Maksudnya gimana Gan?

**Comment by Recehan-Slayer** [comment]
`shrink.*##+js(set, blurred, false)`
Jdi biasanya kan ada countdown serverside di bnyak shorteners dan waktunya berbeda" ada yg cmn sebentar 3detik bahkan ada yg sampai 30detik. Nah pas kita pindah tab atau buka yg lain tuh countdown g jalan. Jdi kyk deteksi fokus gtu. Klo counydown yg 30 detik gila aja hrus mntengin tuh waktu g boleh pndah tab dll contoh link. 
http://p4link.com/aGGnBua ubo filternya jgn dihidupin klo mw tes

**Comment by realodix** [comment]
Harusnya sih tidak masuk scope, kalaupun dilakukan pelebaran, harus dimasukkan ke AdBlockID+

**Comment by Recehan-Slayer** [comment]
yg dari h`ttps://shorte.st/`
semua deteksi focus gan

http://gestyy.com/w7FxMx
http://ceesty.com/weC7bY
http://festyy.com/wM2dCY
clkme.me
clkmein.com
cllkme.com
corneey.com
destyy.com
festyy.com
gestyy.com
jnw0.com
sh.st
viid.me
wiid.me
